### DISCUSSION TITLE
TLS_LEVEL

### DISCUSSION THREAD

**Discussion by nickshanks347** [docker-mailserver/docker-mailserver#3448] [Q&A] [answered]
Does [TLS_LEVEL](https://docker-mailserver.github.io/docker-mailserver/edge/config/environment/#tls_level) support TLS1.3 when set to "immediate"? Or is TLS1.3 only supported on "modern"?

Thanks

**Comment by casperklein** [comment, marked as answer]
Both levels support TLS 1.3:

https://github.com/docker-mailserver/docker-mailserver/blob/da984e5696f124373c4098c30a49e4c5a979278d/target/scripts/helpers/ssl.sh#L133-L153

**Reply by nickshanks347** [reply]
Thank you very much... do you have any idea what this would mean?

```
mail_server     | Jul 29 01:00:05 mail postfix/postscreen[7135]: 
NOQUEUE: reject: RCPT from [*137.184.144.158*]:37254: 550 5.5.1 
Protocol error; from=<noreply@bbs.phreaknet.net>, to=<me@example.com>,
proto=ESMTP, helo=<bbs.phreaknet.org>

me@example.com is just an example.... my real email was in there.
```

Seems like it's rejecting because of a protocol error with a code of 5.5.1 but I can't find much about it online.

**Reply by casperklein** [reply]
Nope, I am not using postscreen myself. You can disable it with `POSTSCREEN_ACTION=ignore`.

PS: Does this only happens with this specific sender? The message indicates, that the sending server does not properly speaks "SMTP".
